app/src/examples/DynamicStylesExample.tsx

```diff
@@ -0,0 +1,186 @@
+import { Text, StyleSheet, View, Button } from 'react-native';
+
+import React from 'react';
+
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+export default function DynamicStylesExample() {
+  const [extraStyle, setState] = React.useState(false);
+  const [rerender, setRerender] = React.useState(false);
+  const widthShared = useSharedValue(80);
+  const backgroundColorShared = useSharedValue<string>('rgb(255,127,63)');
+  const transformScaleYShared = useSharedValue(1);
```

Comment:
Let's separate shared state from animated styles by adding a newline here.
```suggestion
  const transformScaleYShared = useSharedValue(1);

```